Add a note about reserved namespace prefix. Resolves #21803 (#21995)
kubernetes/website: content/en/docs/concepts/overview/working-with-objects/namespaces.md

Creation and deletion of namespaces are described in the [Admin Guide documentation
for namespaces](/docs/admin/namespaces).

{{< note >}}

>     Avoid creating namespace with prefix `kube-`, since it is reserved for Kubernetes system namespaces.

Reviewer: ```suggestion
Namespaces with names that start with `kube-` are reserved for Kubernetes system use.
```
